src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -926,7 +926,7 @@ public void testGetFeedbackSessionResultsForUser() throws Exception {
                 "FSRTest.student6InCourse1@gmail.tmt=student6 In Course1",
                 "FSRTest.instr1@course1.tmt=Instructor1 Course1",
                 "FSRTest.instr2@course1.tmt=Instructor2 Course1",
-                "Anonymous student 283462789@@Anonymous student 283462789.com=Anonymous student 283462789",
+                "Anonymous student 1479876785@@Anonymous student 1479876785.com=Anonymous student 1479876785",
```

Comment:
Yes, it is hashcode of the student's name so if the student name is changed, the hashcode should be changed as well.